projects/plugins/jetpack/functions.opengraph.php

```diff
@@ -224,8 +224,9 @@ function jetpack_og_tags() {
 	 *
 	 * @param array $tags Array of Open Graph Meta tags.
 	 * @param array $args Array of image size parameters.
+	 * @param number $id  ID of the current post for the tags
 	 */
-	$tags = apply_filters( 'jetpack_open_graph_tags', $tags, compact( 'image_width', 'image_height' ) );
+	$tags = apply_filters( 'jetpack_open_graph_tags', $tags, compact( 'image_width', 'image_height' ), $data->ID );
```

Comment:
Do we need to add this? Would it be possible to just use `get_post()` or `get_the_ID()` in the filter instead?

Reply:
My core WP knowledge failed me! Good idea.